traffic_ops/traffic_ops_golang/trafficstats/deliveryservice.go

```diff
@@ -341,21 +270,20 @@ func getDSSummary(client *influx.Client, conf *tc.TrafficDSStatsConfig, db strin
 		})
 	ts, err := getSummary(db, q, client)
 	if err != nil || ts == nil {
-		return nil, nil, nil, err
+		return nil, err
 	}
 
-	var totalKB *float64
-	var totalTXN *float64
-	value := float64(ts.Count*60) * ts.Average
+	s.TrafficStatsSummary = *ts
+
+	value := float64(s.Count*60) * s.Average
 	if conf.MetricType == "kbps" {
-		// TotalBytes is actually in units of kB....
-		value /= 8
-		totalKB = &value
+		value /= 1000
+		s.TotalBytes = &value
 	} else {
-		totalTXN = &value
+		s.TotalTransactions = &value
 	}
 
-	return ts, totalKB, totalTXN, nil
+	return &s, nil
 }
 
 func dsTenantIDFromXMLID(xmlid string, tx *sql.Tx) (bool, uint, error) {
```

Comment:
This file should not have been changed

Reply:
See [comment](https://github.com/apache/trafficcontrol/pull/4633#issuecomment-617996513), fixed in 003af41431.